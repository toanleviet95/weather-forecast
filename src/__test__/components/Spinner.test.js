import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '@/components/Spinner';

describe('Spinner', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Spinner />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
