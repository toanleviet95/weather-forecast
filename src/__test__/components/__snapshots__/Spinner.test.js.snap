// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div
  className="sc-bdnylx fIPKQY"
>
  Loading...
</div>
`;
